.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -1,7 +1,7 @@
 name: 🎉 Feature
 description: Suggest a new feature for the Fivetran dbt package
 title: "[Feature] <title>"
-labels: ["enhancement"]
+labels: ["type:enhancement"]
```

Comment:
Similar to the bug-report comment, let's update these to be in line with our current labeling standards.

Reply:
updated